config/packages/validator/validation.yaml

```diff
@@ -238,6 +238,14 @@ Centreon\Domain\Monitoring\ResourceFilter:
                 - Type:
                     type: int
                     groups: [Default]
+        monitoringServerIds:
+            - Type:
+                type: array
+                groups: [Default]
+            - All:
+                - Type:
+                    type: int
+                    groups: [Default]
```

Comment:
seems not used
```suggestion
```

Reply:
It is used after all when validating the ResourceEntity